README.md

```diff
@@ -34,6 +34,7 @@ Try our demo at [https://demo.ragflow.io](https://demo.ragflow.io).
 
 ## 📌 Latest Updates
 
+- 2024-05-28 Supports LLM Baichuan and VolcanoArk.

```

Comment:
```suggestion
- 2024-05-28 Supports LLM Baichuan and Volcano Ark.

```
The most commonly used naming is Volcano Ark: https://technode.com/2023/06/29/bytedances-volcengine-unveils-ai-model-service-platform-volcano-ark/

Reply:
Same for the other two instances.